lms/djangoapps/branding/api.py

```diff
@@ -170,6 +170,47 @@ def _footer_social_links():
     return links
 
 
+def _build_support_form_url(full_path=False):
+    """
+    Return the support form path
+
+    Returns url of support form, which can have 3 possible values:
+    - '' if the contact form is disabled (by setting SiteConfiguration
+       CONTACT_US_PAGE = True), by the wich is ignored by the
+       parent function
+    - The normal edx support form path if CONTACT_US_PAGE is not set
+    - CONTACT_US_PAGE if the the user has set a custom URL redirect
+      for support forms
+
+    Parameters:
+        - full_path: bool. Appends base_url to returned value if
+                     CONTACT_US_PAGE = True
+
+    Returns: string
+
+    """
+    # The contact us page is enabled by default
+    contact_us_conf = configuration_helpers.get_value('CONTACT_US_PAGE', True)
+
+    if contact_us_conf:
+        # If contact_us_conf is bool enable contact form if it's string,
+        # set link to url
+        if isinstance(contact_us_conf, bool):
+            contact_us_page = reverse("support:contact_us")
+
+            if full_path:
+                return '{base_url}{contact_us_path}'.format(
+                    base_url=settings.LMS_ROOT_URL,
+                    contact_us_path=contact_us_page,
+                )
+            return contact_us_page
+
+        else:
+            return contact_us_conf
+
+    return ''
```

Comment:
Is this going to eventually generate a blank link (i.e. a link with `<a>` tags but no content inside)? If so, I think that's an accessibility problem, and we'd need to make sure the link was gone entirely in the output.

Reply:
That's treated on the parent function, were all empty links are shown on the page.
[here](https://github.com/edx/edx-platform/blob/ca262ffe0f98a613f1f0a9d13fec45946e92fb87/lms/djangoapps/branding/api.py#L254) and [here](https://github.com/edx/edx-platform/blob/ca262ffe0f98a613f1f0a9d13fec45946e92fb87/lms/djangoapps/branding/api.py#L230)